### PR TITLE
[Feature #158422650] view join requests 

### PR DESCRIPTION
**Pull request by TheSteelGuy**
**Introduction**
This PR creates a  page where a user can see all the join requests for their rides.
**Description of Task completed:**

- A  table with each with 4 columns, the last two columns allow ride owner to either reject or accept ride join request.

How to manually test

**Clone this repository**

navigate into the repository after cloning
navigate further in the UI directory open the joinrequest.html page with a browser

**ScreenShot**
joinrequests

**After changes**
ajoin

### CODE REVIEW

**Comment by SelaDanti** [comment]
The user interface design looks  amazing. I would advice to use non competing colours i.e green background and green text

**Comment by TheSteelGuy** [comment]
Thanks @SelaDanti  i will sure change it,

**Comment by Xerrex** [comment]
Maybe for the colors checkout https://coolors.co/. good work though

File: index.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+                <h4>SIGNUP <a href="UI/signup.html" style="text-decoration: none;">Here</a></h4>
+                <hr>
+            </div>
+        </div>
+        <div id="footer">
+            <p>&copy Ridemyway.All rights reserved </p>
+        </div>
+    </div><!--end of wrapper-->
+</body>
+</html>
```
Comment (AnthonyGW) [inline index.html]: Add a newline at the end of the file

File: UI/joinrequest.html [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+                      </tr>
+                </table>
+            </div>
+        </div><!--end body-->
+        <div id="footer">
+            <p>&copy Ridemyway.All rights reserved </p>
+        </div>
+    </div><!--end of join requests-->
+</body>
+</html>
```
Comment (AnthonyGW) [inline UI/joinrequest.html]: Add a newline to the end of the file

File: UI/create_ride.html [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="stylesheet" type="text/css" media="screen" href="main.css" />
+    <script src="main.js"></script>
+    <title>Create Ride</title>
+</head>
+<body>
+    <div id="all_rides"><!--begin all rides-->
+        <div id="header">
+            <ul>
+              <li><a class="active"href="../index.html">Home</a></li>
```
Comment (AnthonyGW) [inline UI/create_ride.html]: Use consistent markup indentation (4 spaces)

**Pull request merged**